README.md

```diff
@@ -1 +1,84 @@
-# personal-finance-back
+# Personal Finance Application
+
+## Description
+
+The Personal Finance Application allows the user to view how their every day expenses and income affect budgeting. 
+
+---
+## Features
+
+* Login
+* Logout
+* Track Balance
+* Update Income
+* Update Expenses
+* Update Balance
+* View Projected Balance
+---
+## Technologies Used
+
+* Java 8
+* PostgreSQL 42
+* Maven 2
+* Tomcat 9
+* JUnit 5
+* Mockito
+* Log4j2
+* Spring Framework 5
+* Spring Boot
+* Spring Data JPA
+* Spring MVC 5
+* Spring AOP
+* Spring Test 5
+* Jackson 2
+* Lombok
+* Firebase Authentication/Authorization 7
+---
+## Getting Started
+
+1. Properly setup the frontend provided in the following `README.md` at:
+
+    > https://github.com/personal-finance-app1/personal-finance-front
+
+2. Copy the repository url from the HTTPS tab by pressing the green clone button, or copy the url and append `.git` to the end.
+
+3. Clone the repository by opening Git Bash at the desired location and running the clone command:
+
+    > git clone (name-of-url).git
+
+4. Configure enivornment variables in `application.properties` to connect to your own PostgreSQL database.
+
+5. Run as a Spring Boot Application.
+
+---
+## Usage
+
+User can login to their account, and they can view their account details after inputting an initial balance. In the homepage screen, they can update their income, expenses, and balance. In the second tab, the user can view their projected balance for the following weeks. The user can end their session by pressing the logout button at the top right of the screen.
+
+---
+## Contributors
+
+Project Lead:
```

Comment:
If you'd like to include team members, please also provide your iteration number and date range when you worked on it.